### PR TITLE
Code-style changes in PR#1248

### PR DESCRIPTION
**Pull request by chinmayapancholi13**
This PR implements some code-style changes in https://github.com/RaRe-Technologies/gensim/pull/1248 as suggested by @piskvorky [here](https://github.com/RaRe-Technologies/gensim/pull/1248#pullrequestreview-41963573).

### CODE REVIEW

File: docs/notebooks/keras_wrapper.ipynb [hunk 1]
```diff
@@ -252,7 +217,9 @@
    "source": [
     "word_a = 'graph'\n",
     "word_b = 'trees'\n",
-    "output = keras_model.predict([np.asarray([model.wv.vocab[word_a].index]), np.asarray([model.wv.vocab[word_b].index])])  # output is the cosine distance between the two words (as a similarity measure)\n",
+    "output = keras_model.predict([np.asarray([model.wv.vocab[word_a].index]), np.asarray([model.wv.vocab[word_b].index])])\n",
```
Comment (menshikh-iv) [inline docs/notebooks/keras_wrapper.ipynb]: Please move comment above this line

Comment (chinmayapancholi13) [inline docs/notebooks/keras_wrapper.ipynb]: Done. Thanks.

**Comment by menshikh-iv** [comment]
Thanks, @chinmayapancholi13.

**Comment by piskvorky** [comment]
@chinmayapancholi13 any reason why `Keras_w2v` is capital `K`?

Variables in Python should be lowercase by convention.

**Comment by chinmayapancholi13** [comment]
@piskvorky Yes! My apologies that I wasn't vigilant enough for this case. I'll fix this in one of the following related PRs.

**Comment by piskvorky** [comment]
No problem :) As long we're converging!

**Pull request merged**